pkg/cmd/kafka/topic/consume/consume.go

```diff
@@ -86,7 +86,7 @@ func NewConsumeTopicCommand(f *factory.Factory) *cobra.Command {
 	flags.Int32Var(&opts.partition, "partition", 0, f.Localizer.MustLocalize("kafka.topic.consume.flag.partition.description"))
 	flags.StringVar(&opts.from, "from", DefaultTimestamp, f.Localizer.MustLocalize("kafka.topic.consume.flag.from.description"))
```

Comment:
Wondering if we need 2 flags for Unix Epoch and Actual Date. 
I would prefer 2 flags for simplicity of documentation and validation

Reply:
I think flag modifying another flag can be hard to work on and confusing. 

I would suggest:

--from-time (ISO 8601 date - short format)
--from-timestamp (epoch/unix timestamp)



I would love to save this dicussion somewhere so we can recall this when building CLI guidelines.

Reply:
Btw. That is my opinion and this is opinionated part so do not feel that you need to do it this way. It is more about my personal preference to keep docs clean.